local-volume/helm/README.md

```diff
@@ -43,3 +43,20 @@ The generated specs can be further customized as needed (usually not necessary),
 
 Default values.yaml is located in local-volume/helm/provisioner folder, user should not remove variables from this file but can
 change any values of these variables.
+
+## Advanced options
+
+### Set engine
+In order to generate the environment specific provisioner's spec, **--set engine={gcepre19,gcepost19,gke,baremetal}** parameter
```

Comment:
Will helm charts continue to provide options to generate backward compatible manifests?

Reply:
Yes, but I would document them under advanced options